tests/unit/libbpf_test.cpp

```diff
@@ -2259,6 +2263,19 @@ TEST_CASE("enumerate link IDs with bpf", "[libbpf]")
     REQUIRE(bpf(BPF_OBJ_GET_INFO_BY_FD, &attr, sizeof(attr)) == 0);
     REQUIRE(info.id == id1);
 
+    // Get info on the second link.
+    memset(&attr, 0, sizeof(attr));
+    info = {};
+    attr.info.bpf_fd = fd2;
+    attr.info.info = (uintptr_t)&info;
+    attr.info.info_len = sizeof(info);
+    REQUIRE(bpf(BPF_OBJ_GET_INFO_BY_FD, &attr, sizeof(attr)) == 0);
+    // TODO: Should this be BPF_LINK_TYPE_PLAIN?
```

Comment:
As far as I know, yes.
```suggestion
    // TODO: This should be BPF_LINK_TYPE_PLAIN.
```

Reply:
See https://github.com/microsoft/ebpf-for-windows/issues/4096